src/components/campaigns/Summary/Goal/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import numbro from 'numbro'
+import Stat from '../Stat'
+
+const Goal = ({ loading, goal, currencySymbol, numberFormat }) => {
+  if (!goal) return <span />
+  const formattedGoal = numbro(goal).format(currencySymbol + numberFormat)
+
+  return (
+    <Stat
+      loading={loading}
+      number={formattedGoal} />
+  )
+}
+
+Goal.displayName = 'CampaignSummaryGoal'
+
+Goal.propTypes = {
+  amount: React.PropTypes.string,
```

Comment:
This prop type doesn't appear to be required – maybe it's supposed to be `goal` now?